src/main/java/org/apache/sysml/utils/Statistics.java

```diff
@@ -112,6 +113,17 @@ public static synchronized long getNoOfExecutedMRJobs() {
 		return numExecutedMRJobs.longValue();
 	}
 	
+	private static AtomicLong numNativeFailures = new AtomicLong(0);
+	public static AtomicLong numNativeLibMatrixMultCalls = new AtomicLong(0);
+	public static AtomicLong numNativeLibMatrixDNNCalls = new AtomicLong(0);
```

Comment:
Use LongAdder instead of AtomicLong - the former internally avoids contention by using multiple partial results.